ui/src/segments/details/SegmentDetailsView.js

```diff
@@ -62,49 +65,48 @@ const SegmentDetailsView = ({ projectId, segmentId, ...props }) => {
   return (
     <EuiPageTemplate restrictWidth={restrictWidth} paddingSize={paddingSize}>
       <EuiSpacer size="l" />
-        {!isLoaded ? (
-          <EuiTextAlign textAlign="center">
-            <EuiLoadingChart size="xl" mono />
-          </EuiTextAlign>
-        ) : error ? (
-          <EuiCallOut
-            title="Sorry, there was an error"
-            color="danger"
-            iconType="alert">
-            <p>{error.message}</p>
-          </EuiCallOut>
-        ) : (
-          <Fragment>
-            {!(props["*"] === "edit") && (
-              <Fragment>
-                <EuiPageTemplate.Header
-                  bottomBorder={false}
-                  pageTitle={<PageTitle title={data.data.name} />}
-                >
-                  <SegmentActions
-                    onEdit={() => props.navigate("./edit")}
-                    onDeleteSuccess={() => props.navigate("../")}>
-                    {(getActions) => (
-                      <PageNavigation
-                        tabs={tabs}
-                        actions={getActions(data.data)}
-                        selectedTab={props["*"]}
-                        {...props}
-                      />
-                    )}
-                  </SegmentActions>
-                </EuiPageTemplate.Header>
-              </Fragment>
-            )}
+      {!isLoaded ? (
```

Comment:
Minor nitpick: should we indent this block? Or rather in general do we prefer to indent such blocks when they are enclosed in tags?

Reply:
I don't have a strong preference on this and from the search that I did, though ESlint and other formatters seem to support some configurations to enforce this, the general guideline is to keep it readable. So I suppose we could leave it as it is. Unless you have some other suggestions to bring about more consistency.

Reply:
Oh I don't really have a preference really, it's just that I thought I spotted another change made in this PR where a similar block was indented (for the reverse reason) 😅 I guess we can keep it as it is and then... fix some convention if it gets too inconsistent that we get annoyed by it 😶 Thanks a lot for looking up all the formatters though!